components/mdc/Menu/Menu.svelte

```diff
@@ -0,0 +1,63 @@
+<!-- https://github.com/material-components/material-components-web/tree/master/packages/mdc-menu -->
+<script>
+import { onDestroy, onMount } from 'svelte'
```

Comment:
We try to alphabetize our imports by where they're imported from (ignoring .. path segments) and putting our in-app imports before npm-dependency imports. Looks like I missed this one.

Reply:
```suggestion
import { onMount } from 'svelte'
```